src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Confirm.cshtml

```diff
@@ -84,6 +84,18 @@
     </div>
 </div>
 
+<form method="post">
+
+    @Html.AntiForgeryToken()
+    @Html.HiddenFor(x => x.EmployerAccountLegalEntityPublicHashedId)
+    @Html.HiddenFor(x => x.NewPrice)
+    @Html.HiddenFor(x => x.NewStartDate)
+    @Html.HiddenFor(x => x.ApprenticeshipHashedId)
+
+    <button type="Submit" class="govuk-button">Save and send</button>
```

Comment:
Id may help automation testing. But as this is the only button on the page, I will leave it to you to decide.

Reply:
Quite right.